lib/board.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+
+class Board
+	attr_accessor :cells
+
+	def initialize
+		self.reset!
+	end
+
+	def cells
+		@cells
+	end
+
+	def reset!
+		self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+	def display
+		puts " #{self.cells[0]} | #{self.cells[1]} | #{self.cells[2]} "
+		puts "-----------"
+		puts " #{self.cells[3]} | #{self.cells[4]} | #{self.cells[5]} "
+		puts "-----------"
+		puts " #{self.cells[6]} | #{self.cells[7]} | #{self.cells[8]} "
+	end
+
+	#Returns value at that position
+	def position(user_input)
+		index = (user_input.to_i-1)
+		self.cells[index]
+	end
+
+	def full?
```

Comment:
Use the most expressive and efficient iterator for a problem. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`cells.all? {|cell| cell != " "}`

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.